src/createServer.js

```diff
@@ -1,9 +1,27 @@
 /* eslint-disable no-console */
 'use strict';
 
+const http = require('http');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-type', 'application/json');
+
+    const formattedUrl = req.url.replace(/\/+/g, '/');
+    const normalizedUrl = new URL(formattedUrl, `http://${req.headers.host}`);
+
+    const parts = normalizedUrl.pathname.slice(1).split('/');
+    const query = Object.fromEntries(normalizedUrl.searchParams);
```

Comment:
Let's handle a case where there are multiple values for one param

Reply:
Make sense, handled.